public/app/app.bootstrap.js

```diff
@@ -80,6 +86,7 @@
     }).then(function(results) {
       collabosphere.constant('me', results.me.data);
       collabosphere.constant('config', results.config.data);
+      collabosphere.constant('deepLinkId', parameters.deep_link_id);
```

Comment:
@paulkerschen In my local testing, this block of code is hit with every request. I'd like to believe we have no risk persist the attribute across session. Is there an edge case here?